### PR TITLE
Fixed an issue where app center wasn't logging events

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

File: Files/ViewModels/SettingsViewModel.cs [hunk 1]
```diff
@@ -50,6 +53,7 @@ private async Task<SettingsViewModel> Initialize()
             FileTagsSettings = new FileTagsSettings();
 
             // Send analytics to AppCenter
+            StartAppCenter();
```
Comment (gave92) [inline Files/ViewModels/SettingsViewModel.cs:56]: If the purpose of this change is to make sure that `AppCenter.Start()` is called before `TrackAnalytics()` note that the `async void` method will return before reaching it.

Comment (yaichenbaum) [reply Files/ViewModels/SettingsViewModel.cs:56]: Would `await` solve this issue?

Comment (gave92) [reply Files/ViewModels/SettingsViewModel.cs:56]: Yup

**Review by winston-de — APPROVED** [review]

**Comment by gave92** [comment]
Uff perhaps I was wrong, adding await means that the app will wait for AppCenter before continuing startup. Probably the difference is not noticeable though and it could even be the right thing to do 🤔

**Comment by yaichenbaum** [comment]
> Uff perhaps I was wrong, adding await means that the app will wait for AppCenter before continuing startup. Probably the difference is not noticeable though and it could even be the right thing to do 🤔

Hmm, I have a different idea.

**Pull request merged**